### ISSUE TITLE
Can not handle its vodka

### ISSUE THREAD

**Issue by Nicholas-Baron** [itinerarium/phoneme-synthesis#5]
Tried to have it pronounce "wszystko" 
[https://en.wiktionary.org/wiki/wszystko](https://en.wiktionary.org/wiki/wszystko)

Input copied from website: /ˈfʂɨ.stkɔ/

It can say the first sound, but not any other.

**Comment by savetier** [comment]
Try ˈfʃɨstkɔ

**Comment by Nicholas-Baron** [comment]
It works, but from my hobbyist understanding of phonetics, /ʂ/ is not the same as /ʃ/. It may be close enough, but asking the user to change one sound into another feels "wrong".

**Comment by savetier** [comment]
You are right but the phoneme synthesis can't handle a lot of sounds, unfortunately.

**Comment by Nicholas-Baron** [comment]
Maybe detecting a sound that causes the output to be silent and suggesting "close enough" substitutions would alleviate the problem.
